rules/integrations/azure/initial_access_entra_illicit_consent_grant_via_registered_application.toml

```
@@ -13,7 +13,7 @@ accomplished by tricking a user into granting consent to the application, typica
 establishes an OAuth grant that allows the malicious client applocation to access resources on-behalf-of the user.
 """
 from = "now-9m"
-index = ["logs-azure*"]
+index = ["logs-azure.auditlogs*"]
```

```suggestion
index = ["logs-azure.auditlogs-*"]
```
only because you include the dash in your other auditlogs rules